pegasus/sites.v3/hourofcode.com/layouts/default.haml

```diff
@@ -2,7 +2,7 @@
 
 .wrapper
   = view :unsupported_browser
-  - if request.cookies['pm'] == 'new_header'
+  - if request.cookies['pm'] == 'new_header' && request.site == 'code.org'
```

Comment:
Actually I will remove new_header support from this file since hourofcode.com should never show it.